### ISSUE TITLE
Hook unit tests into test_suite in setup.py

### ISSUE THREAD

**Issue by koobs** [mozilla/bleach#97]
Allow packagers/porters to run via standard python setup.py test command:

```
--- ./setup.py.orig     2013-06-10 11:19:04.719234114 +1000
+++ ./setup.py  2013-06-10 11:19:34.305595246 +1000
@@ -23,5 +23,6 @@
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Topic :: Software Development :: Libraries :: Python Modules',
-    ]
+    ],
+    test_suite='nose.collector'
 )
```

**Comment by willkg** [comment]
I never found this particularly helpful. I'm going to close this out.

If someone has a compelling reason why this is interesting, I'm game for rethinking it.

**Comment by koobs** [comment]
A working 'test' command is good packaging practice and the canonical method for doing so. It enables people downstream (users, packagers, whoever) to have tests 'just work' without having to hunt/search for <some-custom-test-invocation-command-dejour> on a website (if it exists at all). 

Different 'test runners' employ/recommend different methods for this integration. For nose it's this. See also:

nose: http://nose.readthedocs.io/en/latest/setuptools_integration.html
pytest: http://pytest.org/latest/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner

**Comment by koobs** [comment]
Please re-open and merge.

**Comment by willkg** [comment]
When I was packaging, we asked projects to do Makefiles. There are a lot of cases where `python setup.py test` isn't flexible or sufficient.

This isn't a PR, so there's nothing to merge here. I'll reopen and implement it.

**Comment by willkg** [comment]
Fixed in #232.
